backend/PythonClient/multirotor/monitor/point_deviation_monitor.py

```diff
@@ -1,10 +1,14 @@
 # sUAS shall not deviate from their planned routes by more than [10%] of the total distance.
 from time import sleep
+from datetime import datetime
+import threading
 
 from PythonClient.multirotor.util.geo.geo_util import GeoUtil
 from PythonClient.multirotor.util.graph.three_dimensional_grapher import ThreeDimensionalGrapher
 from PythonClient.multirotor.monitor.abstract.single_drone_mission_monitor import SingleDroneMissionMonitor
+from PythonClient.multirotor.airsim_application import AirSimApplication
```

Comment:
This is redundant, `PointDeviationMonitor(SingleDroneMissionMonitor)` already inherits from AirSimApplication.